servers/jax-rs-tests/pom.xml

```diff
@@ -124,6 +124,11 @@
       <artifactId>logback-classic</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.apache.iceberg</groupId>
+      <artifactId>iceberg-spark3-runtime</artifactId>
+      <scope>provided</scope>
```

Comment:
Why is this a non-test dependency - i.e. Iceberg leaking to other modules.

Reply:
for spark configuration of `spark.sql.catalog.nessie.catalog-impl`, I have used `NessieCatalog.class.getName()`.
Hence it couldn't be a test dependency. 

But now I changed conf to use `org.apache.iceberg.nessie.NessieCatalog`.
**Hence changed now to test dependency.**